pkg/controllers/smart.go

```diff
@@ -2,15 +2,23 @@ package controllers
 
 import (
 	"encoding/json"
+	"net/http"
+
 	"github.com/dghubble/go-twitter/twitter"
 	"github.com/smartatransit/third_rail/pkg/clients"
-	"net/http"
 )
 
 type SmartController struct {
 	TwitterClient clients.TwitterClient
 }
 
+// GetParkingStatus godoc
+// @Summary Get Parking Information
+// @Description  Get available parking information as informed by twitter
+// @Produce  json
+// @Success 200 {object} parkingResponse
+// @Router /smart/parking [get]
+// @Security ApiKeyAuth
```

Comment:
I noticed that some of the other endpoints on this file aren't exposed yet, so I didn't annotate them